lib/Bio/EnsEMBL/DataCheck/Checks/CheckNonMinimisedGATs.pm

```diff
@@ -0,0 +1,80 @@
+=head1 LICENSE
+
+Copyright [2018-2021] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::DataCheck::Checks::CheckNonMinimisedGenAlnTrees;
```

Comment:
Module and package names don't have to be the same, but unless there's a good reason not to, I think it's best if the package name matches the file name. (The index.json file will need to be updated accordingly if this suggestion is accepted.)
```suggestion
package Bio::EnsEMBL::DataCheck::Checks::CheckNonMinimisedGATs;
```

Reply:
Ah yes, this was an oversight when I renamed the datacheck